modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES
+----
+
+.Result
+[role="queryresult"]
+----
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name     | type       | aliases                      | access       | address          | role      | writer | requestedStatus | currentStatus | statusMessage | default | home  | constituents |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "movies" | "standard" | ["films", "motion pictures"] | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []           |
+| "neo4j"  | "standard" | []                           | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | TRUE    | TRUE  | []           |
+| "system" | "system"   | []                           | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []           |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+The results of this command are filtered according to the `ACCESS` privileges of the user.
+However, some privileges enable users to see additional databases regardless of their `ACCESS` privileges:
+
+* Users with `CREATE/DROP/ALTER DATABASE` or `SET DATABASE ACCESS` privileges can see all standard databases.
+* Users with `CREATE/DROP COMPOSITE DATABASE` or `COMPOSITE DATABASE MANAGEMENT` privileges can see all composite databases.
+* Users with `DATABASE MANAGEMENT` privilege can see all databases.
+
+If a user has not been granted `ACCESS` privilege to any databases nor any of the above special cases, the command can still be executed but will only return the `system` database, which is always visible.
+
+
+[NOTE]
+====
+As of Neo4j 5.3, databases hosted on servers that are offline are also returned by the `SHOW DATABASES` command.
+For such databases, the `address` column displays `NULL`, the `currentStatus` column displays `unknown`, and the `statusMessage` displays `Server is unavailable`.
+====
+
+=== Show detailed information for a particular database
+
+In this example, the detailed information for a particular database can be displayed using the command `SHOW DATABASE name YIELD *`.
+When a `YIELD` clause is provided, the full set of columns is returned.
+
+.Query
+[source, cypher, role=test-result-skip]
+----
+SHOW DATABASE movies YIELD *
+----
+
+// Limited result set.
+// The output has been capped.
```

Comment:
The reason we initially capped it was cause it got to big to properly read when printed in the manual, let's confirm in the preview that the new format does not share this limitation

@Hunterness

Reply:
Having the format in cypher-shell means that readers will have to scroll the code block to see the full example.
This may not be ideal. However, it is more accurate and solves the problem we currently face elsewhere of very narrow table columns (e.g. https://neo4j.com/docs/cypher-manual/current/administration/databases/).

Reply:
scrolling is still preferable to the squished columns 👍